Missing iterator_pair.h for building mymodule in nest 2.20.0

**Describe the bug**
When trying to make external-module after nest have being installed, an error message stating that iterator_pair.h was not found is displayed. In addition, copying the file found in libnestutil folder results in another error stating that ../nestkernel/source.h was not found.

**To Reproduce**
Steps to reproduce the behavior:
1. Install NEST release 2.20.0 without external module such as:
```
$ mkdir nest-make-dir
$ cd nest-make-dir
$ cmake -DCMAKE_INSTALL_PREFIX:PATH=<NEST_INSTALL_DIR> -Dwith-mpi=ON -Dwith-python=3 -DPYTHON_EXECUTABLE=`which python3.6` -DNOSETESTS=`which nosetests` -DCYTHON_EXECUTABLE=`which cython` -Dwith-boost=ON -Dwith-includes=<BOOST_INCLUDE_PATH> ../nest-simulator-2.20.0
$ source <NEST_INSTALL_DIR>/bin/nest_vars.sh
```
2. Try to install mymodule with:
```
$ cd ..
$ mkdir mymodule-make-dir
$ cd mymodule-make-dir
$ cmake ./nest-simualtor-2.20.0/example/MyModule
$ make
```
3. Output example-1:
```
from nest-simulator-2.20.0/examples/MyModule/mymodule.cpp:29:
<NEST_INSTALL_DIR>/include/nest/sort.h:37:10: fatal error: iterator_pair.h: No such file or directory
#include "iterator_pair.h"
```
4. Copy libnestutil/iterator_pair.h into <NEST_INSTALL_DIR>/include/nest and try to make mymodule again
```
$ cp -a nest-simulator-2.20.0/libnestutil/iterator_pair.h <NEST_INSTALL_DIR>/include/nest
$ make
```
5. Output example-2:
```
<NEST_INSTALL_DIR>/include/nest/iterator_pair.h:29:10: fatal error: ../nestkernel/source.h: No such file or directory
#include "../nestkernel/source.h"
```

**Expected behavior**
Expected that make command ends with no error

**Desktop/Environment (please complete the following information):**
 - OS: CentOS 7.7.1908
 - Shell: bash
 - Python-Version: Python 3.6
 - NEST-Version: nest-2.20
 - Installation: from source, see above (with MPI, with boost)
    Configuration summary not available as it has passed several weeks after NEST installation.

**Additional context**
Changing line 29 in the copied iterator_pair.h to:
```
#include "source.h"
```
Fix the issue!
Should this be included into 2.20.1 (#1455)?

The files mentioned in this issue seems to be fixed in the github master, however, trying to make mymodule after NEST have being installed leads to another error: SFML/Audio.hpp: No such file or directory.

--- a/libnestutil/CMakeLists.txt
+++ b/libnestutil/CMakeLists.txt
@@ -21,6 +21,7 @@ set( nestutil_sources
     block_vector.h
     compose.hpp
     enum_bitfield.h
+    iterator_pair.h
     lockptr.h
     logging_event.h logging_event.cpp
     logging.h

--- a/libnestutil/iterator_pair.h
+++ b/libnestutil/iterator_pair.h
@@ -26,7 +26,7 @@
 #include <boost/iterator/iterator_adaptor.hpp>
 #include <boost/tuple/tuple.hpp>
 
-#include "../nestkernel/source.h"
+#include "source.h"
 
 namespace boost
 {

--- a/nestkernel/CMakeLists.txt
+++ b/nestkernel/CMakeLists.txt
@@ -106,6 +106,7 @@ target_include_directories( nestkernel PRIVATE
     ${PROJECT_BINARY_DIR}/libnestutil
     ${PROJECT_SOURCE_DIR}/librandom
     ${PROJECT_SOURCE_DIR}/sli
+    ${PROJECT_SOURCE_DIR}/nestkernel
     )
 
 install( TARGETS nestkernel

